app/api/dao/user.py

```diff
@@ -188,3 +189,53 @@ def authenticate(username_or_email, password):
             return user
 
         return None
+
+    @staticmethod
+    def get_achievements(user_id):
+        user = UserModel.find_by_id(user_id)
+        all_relations = user.mentor_relations + user.mentee_relations
+        tasks = []
+        for relation in all_relations:
+            tasks += relation.tasks_list.tasks
+        achievements = list(filter(lambda x: x.get("is_done"), tasks))
```

Comment:
Better to use list comprehensions over lambdas and map/reduce/filter.

Reply:
@haroldadmin can you please fix this line according to @ramitsawhney27 's suggestion? I had to do a [quick search](https://python-3-patterns-idioms-test.readthedocs.io/en/latest/Comprehensions.html), but this seems to make sense.

Reply:
Okay, I'll change this to a list comprehension.